### PR TITLE
Update idle-session-timeout-web-apps.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated per Namit's advise

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9107

File: microsoft-365/admin/manage/idle-session-timeout-web-apps.md [hunk 1]
```diff
@@ -71,8 +71,13 @@ When a user has been inactive in Microsoft 365 web apps for the time period you
...
-    - If they're on a managed device (one that is compliant or joined to a domain) and using a supported browser like Microsoft Edge or Google Chrome (with the [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji)). For this feature to trigger on an unmanaged device, an eligible Azure AD Premium P1 or P2 subscription, and a specific Conditional Access policy, is required. See below for further details.
+    - If they get single sign-on (SSO) into the web app from the device joined account
+    - If they selected Stay signed in at the time of sign-in. For more info on hiding this option for your organization, see [Add branding to your organization's sign-in page](/azure/active-directory/fundamentals/customize-branding).
+    - If they're on a managed device (one that is compliant or joined to a domain) and using a supported browser like Microsoft Edge or Google Chrome (with the [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji))
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/idle-session-timeout-web-apps.md]:
```suggestion
    - If they get single sign-on (SSO) into the web app from the device joined account.
    - If they selected **Stay signed in** at the time of sign-in. For more info on hiding this option for your organization, see [Add branding to your organization's sign-in page](/azure/active-directory/fundamentals/customize-branding).
    - If they're on a managed device (one that is compliant or joined to a domain) and using a supported browser like Microsoft Edge or Google Chrome (with the [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji)).
```

File: microsoft-365/admin/manage/idle-session-timeout-web-apps.md [hunk 2]
```diff
@@ -71,8 +71,13 @@ When a user has been inactive in Microsoft 365 web apps for the time period you
...
+
+## Trigger idle session timeout only on unmanaged devices
+
+By default, idle session timeout feature triggers on all device types if the other conditions are met. For this feature to trigger only on an unmanaged device, an eligible Azure AD Premium P1 or P2 subscription is required. You'll also need to add a Conditional Access policy in the Azure AD admin center.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/idle-session-timeout-web-apps.md]:
```suggestion
By default, the idle session timeout feature triggers on all device types if the other conditions are met. For this feature to trigger only on an unmanaged device, an eligible Azure AD Premium P1 or P2 subscription is required. You'll also need to add a Conditional Access policy in the Azure AD admin center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**